From 85fa612554ed84cd6aeb2777212d12c3f5521337 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Tue, 21 Jun 2022 09:20:07 +0800
Subject: [PATCH 48/63] arm64: dts: rockchip: Add mmc aliases for rk3588.dtsi

With this series, adjust MMC alias and see mmcblk id change.
This patch is still useful on ChromeOS.

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Change-Id: I384e49606c5623a403c93e62e9e7165b61e4ca80
---
 arch/arm64/boot/dts/rockchip/rk3588.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588.dtsi b/arch/arm64/boot/dts/rockchip/rk3588.dtsi
index dd1b66176e5..5ff406ac2bc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3588.dtsi
@@ -9,6 +9,9 @@
 / {
 	aliases {
 		ethernet0 = &gmac0;
+		mmc0 = &sdhci;
+		mmc1 = &sdmmc;
+		mmc2 = &sdio;
 	};
 
 	gmac0: ethernet@fe1b0000 {
-- 
2.36.1

